### PR TITLE
Bov Shuffle Silver

### PR DESCRIPTION
**Pull request by jessechoe10**

### CODE REVIEW

File: content/3_Silver/Func_Graphs.mdx [hunk 1]
```diff
@@ -39,7 +39,8 @@ export const problems = {
 			'Easy',
 			false,
 			['Func Graph'],
-			'Try to solve the problem in $mathcal{O}(N)$!'
+			'usaco-764',
+			'Try to solve the problem in $\mathcal{O}(N)$!'
```
Comment (bqi343) [inline content/3_Silver/Func_Graphs.mdx:43]: > 'Try to solve the problem in $\mathcal{O}(N)$!'

remove this line and add the time complexity to the solution

**Review by bqi343 — CHANGES_REQUESTED** [review]

File: solutions/USACO dec17-the-bovine-shuffle-silver.mdx [hunk 2]
```diff
@@ -0,0 +1,84 @@
+---
+id: usaco-764
+source: USACO Silver 2017 December
+title: The Bovine Shuffle
+author: Jesse Choe
+---
+
+[Official Analysis](http://www.usaco.org/current/data/sol_shuffle_silver_dec17.html)
+
+Time Complexity: $\mathcal{O}(N)$
```
Comment (bqi343) [inline solutions/USACO dec17-the-bovine-shuffle-silver.mdx:10]: time complexity should be bolded (as it is in the example)

**Review by bqi343 — CHANGES_REQUESTED** [review]

**Comment by frodakcin** [comment]
I think it's good after you address Ben's comment

File: solutions/USACO dec17-the-bovine-shuffle-silver.mdx (hunk 2 above)
Comment (jessechoe10) [reply solutions/USACO dec17-the-bovine-shuffle-silver.mdx:10]: Isn't it already bolded? I just copied and pasted that from the example.

Comment (bqi343) [reply solutions/USACO dec17-the-bovine-shuffle-silver.mdx:10]: No.
Screen Shot 2021-02-10 at 10 15 27 AM

**Pull request merged**